cmd/server/assets/login/select-password.html

```diff
@@ -80,9 +121,91 @@
         let email = $email.val();
         let pwd = $password.val();
         if (pwd != $retype.val()) {
-          flash("Password and retyped passwords must match.", "danger");
+           flash.error("Password and retyped passwords must match.");
+        return;
+        }
+
+        {{if .requirements}}
+        let upper = 0
```

Comment:
Semicolons, throughout.

Reply:
silly languages that require statement terminators :)

Reply:
The worst part is that it's not "required". It's "interpreted differently depending on the runtime engine"